AK/SIMDMath.h

```diff
@@ -38,6 +38,18 @@ ALWAYS_INLINE static f32x4 frac_int_range(f32x4 v)
     return v - floor_int_range(v);
 }
 
+template<class T>
+ALWAYS_INLINE T bitselect(T v1, T v2, T control_mask)
+{
+    return (v1 & control_mask) | (v2 & ~control_mask);
+}
+
+template<class T>
+ALWAYS_INLINE T abs(T x)
```

Comment:
Nit: Ideally these two templates would be restricted to simd types

Reply:
Do we have something for that?

Reply:
We do, `SIMDVector`. Done.